Makefile

```diff
@@ -185,6 +185,22 @@ generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 		output:rbac:dir=$(RBAC_ROOT) \
 		rbac:roleName=manager-role
 
+## --------------------------------------
+## Tilt / Kind
+## --------------------------------------
+
+.PHONY: kind-create-capz
+kind-create-capz: ## create capz kind cluster if needed
+	./scripts/kind-with-registry.sh
+
+.PHONY: tilt-up
+tilt-up: kind-create-capz ## start tilt and build kind cluster if needed
+	tilt up
+
+.PHONY: kind-delete-capz
+kind-delete-capz: ## delete capz kind cluster
```

Comment:
this overlaps with `make kind-reset` 

should we add `kind delete cluster --name capz` to the `kind-reset` target?